packages/jest-cli/src/search_source.js

```diff
@@ -209,14 +209,22 @@ export default class SearchSource {
     } else if (globalConfig.findRelatedTests && paths && paths.length) {
       return Promise.resolve(this.findRelatedTestsFromPattern(paths));
     } else {
+      const allFiles = new Set(this._context.hasteFS.getAllFiles());
       const validTestPaths =
         paths &&
         paths.filter(name => {
           try {
-            return fs.lstatSync(name).isFile();
+            if (!fs.lstatSync(name).isFile()) {
+              // It exists, but it is not a file; return false.
```

Comment:
can you remove the "return false" here? It's literally the next line.

Reply:
huh, I didn't spot that :D